webauthn/webauthn.py

```diff
@@ -184,8 +186,15 @@ def json(self):
 
 
 class WebAuthnUser(object):
-    def __init__(self, user_id, username, display_name, icon_url,
-                 credential_id, public_key, sign_count, rp_id):
+    def __init__(self, user_id=None, username=None, display_name=None, icon_url=None,
```

Comment:
Hi @mdedonno1337,

Can we get rid of the changes to the constructor here?

Reply:
Basically, just leave the old `def __init__(...)` method signature.

Reply:
Well, with this modification, the PR is probable useless since the repeated None will be mendatory in the `WebAuthnUser` creation.